cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java

```diff
@@ -38,4 +48,18 @@ private static String formatQueryParameters(Map<String, String> queryParams) {
                 .map(entry -> String.format(PARAMETER_KEY_VALUE_PAIR_TEMPLATE, entry.getKey(), entry.getValue()))
                 .collect(Collectors.joining(PARAMETER_DELIMITER));
     }
+
+
+    public static URI getNvaApiId(String identifier) {
```

Comment:
Is `getNvaApiId` a correct name for somethings which add `ORGANIZATION_PATH` to path? Maybe add `String type` to parameters?

Reply:
It constructs a new uri against the NVA api, and adds the string containing four numbers (identifier) -> id